### PR TITLE
Update microos_getting_started.md

### PR DESCRIPTION
**Pull request by syds-git**
added encryption manual, cleaned up layout some more, added commands to make zypper dup autorefresh manually added repos, removed recommend packages that are now included in the pattern

### CODE REVIEW

File: project/docs/microos_getting_started.md [hunk 1]
```diff
@@ -65,18 +102,27 @@ $ sudo rebootmgrctl status
...
+$ sudo reboot
 ```
 Or
 ```
 $ sudo transactional-update shell
   # zypper dup
   # exit
+$ sudo reboot
+```
+if you have added repositories to zypper by yourself, changes are that they are not refreshing automatically with a `dup` command. You'll have to give them the flag to autorefresh:
```
Comment (dfaggioli) [inline project/docs/microos_getting_started.md:114]: "chances are", maybe?

File: project/docs/microos_getting_started.md [hunk 2]
```diff
@@ -65,18 +102,27 @@ $ sudo rebootmgrctl status
...
+  # zypper mr -f <number of repo you want to add Yes to in the Refresh colomn>
+  # exit
+$ sudo reboot
 ```
 
 I also recommend installing a few more packages that are useful.
 ```
-$ sudo transactional-update pkg install
-bash-completion wget nfs-client autofs
+$ sudo transactional-update pkg install nfs-client autofs
```
Comment (dfaggioli) [inline project/docs/microos_getting_started.md:125]: I'm not sure it's really necessary to mention these two. In fact, they're only useful if you want to mount NFS shares over the network (and, as far as `autofs` is concerned, do that automatically).

It's possible that one needs to do that (e.g., I do), but not that common I would say. And they are not needed for things like mounting USB keys or external hard drives (see email in the kubic ML).

I think we should not bother recommending them.

**Review by dfaggioli — CHANGES_REQUESTED** [review]
Nice change! I made a couple of comments, but apart from those, looks good to me.

File: project/docs/microos_getting_started.md (hunk 1 above)
Comment (syds-git) [reply project/docs/microos_getting_started.md:114]: yeah should be maybe. I was also thinking to probably add it to the add OBS packages part as that's where it comes in handy.

File: project/docs/microos_getting_started.md (hunk 2 above)
Comment (syds-git) [reply project/docs/microos_getting_started.md:125]: so when you say network shares, is autofs also the package that automounts my nextcloud webdav? or is that unrelated?

**Comment by Nycticorax** [comment]
Comments on https://github.com/openSUSE/openSUSE-docs-revamped/pull/65/commits/c2e488b64556d2c883826d208671e55e3e22afe7

**Comment by Nycticorax** [comment]
I swear to God I had half a page of comments which I append to my comment above, but for some reason it didn't register and I stupidly didn't save them on my drive. I'll try to fight my bitterness with a coffee. Off the top of my head:
- please clarifiy if the present document presupposes microOS TW or microOS Leap or does not presuppose anything, and then adjust line 182 accordingly;
- let's try to stick to the `style` section on the README
- please clarify what you mean by a "transaction-update shell" for the new user
- please clarify what you mean by "Make sure you install a browser and unzip via transactional-updates if you want to use GNOME extensions". I guess it means that the installing should be done via a transactional update, but let's make it easy on people's brain

**Comment by syds-git** [comment]
> I swear to God I had half a page of comments which I append to my comment above, but for some reason it didn't register and I stupidly didn't save them on my drive. I'll try to fight my bitterness with a coffee. Off the top of my head:
> 
>     * please clarifiy if the present document presupposes microOS TW or microOS Leap or does not presuppose anything, and then adjust line 182 accordingly;
Done

>     * let's try to stick to the `style` section on the README
Not sure what you mean :)

>     * please clarify what you mean by a "transaction-update shell" for the new user
Will this not be part of the portal wiki for MicroOS. I assume we would explain that somewhere else as well or do we need to explain this again?

>     * please clarify what you mean by "Make sure you install a browser and unzip via transactional-updates if you want to use GNOME extensions". I guess it means that the installing should be done via a transactional update, but let's make it easy on people's brain
Explained

**Pull request merged**